CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] A new ledger era is enabled on the Cardano mainnet, which implements the above specification. This will be split into two stages, as described above.
+
+### Implementation Plan
+
+This proposal restricts itself to the core on-chain mechanism. However, several issues remain to be discussed. These include:
+
+1. off-chain DRep expectations;
+2. bootstrapping the Constitutional Committee;
+3. the initial Constitution;
+4. DRep incentives;
+5. which values in the [ratification requirement table](#requirements) should be hard-coded, and which should be protocol parameters?
+
+Some of these issues may be addressed by changes to this document; others will need to be addressed in new CIPs or discussion documents.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
+
+* The `PPUP` transition rule will be rewritten and moved out of the `UTxO` rule and into the `LEDGER` rule as a new `TALLY` rule.
+
+  It will process the governance actions and the votes, ratify them, and stage governance actions for enactment in the current or next epoch, as appropriate.
+
+* The `NEWEPOCH` transition rule will be modified.
+* The `MIR` sub-rule will be removed.
+* A new `ENACTMENT` rule will be called immediately after the `EPOCH` rule. This rule will enact governance actions that have previously been ratified.
+* The `EPOCH` rule will no longer call the `NEWPP` sub-rule or compute whether the quorum is met on the PPUP state.
+
+#### More on DRep incentives
+
+The DReps arguably need to be compensated for their work.
+
+The corresponding incentive mechanisms need to be specified, with the funds probably coming from the per-epoch treasury allocation.
+Performance constraints will also need to be considered since it would be problematic if millions of DReps were expected to vote on each governance action. Some incentive options to ensure a manageable number of DReps include:
```

Comment:
I thought the whole point of DReps was that most votes should be delegated, thus avoiding the problem of millions of people wanting to vote? That suggests that setting up incentives to encourage delegation is pretty important.

If everyone just votes themselves then I would say the DRep system has failed!

Reply:
I thought the point of DReps was to enable liquid democracy where-by individual experts can vote directly and people who don't want to devote the time or attention can delegate their vote to a trusted representative. 

If a randomized committee is selected per epoch per action weighted by stake, I feel this would feel a lot like disenfranchisement for a lot of smaller DReps and the voting could be criticized as illegitimate.

Reply:
> I thought the whole point of DReps was that most votes should be delegated

I think this is one of the most imporant issues in this text, that it seems to contradict itself in that it wants to incentivise dReps, but I think it will be hard for meaningful representation to happen if everyone representes themselves (for reasons previously stated).

I do not like the idea of randomized subsets of the dReps, since that will really leave the door open for missrepreserntation imo (read earlier comments about this). I would be much more comfortable with dReps on specific classes of issues (read committees). In that case, the total set of representatives/votes/votes will not be as large for any given action/vote (since only a subset of dReps will vote on any class of issues - the ones they a representatives of).

Reply:
This is the topic that is the most uncertain at this point, namely how to limit the DReps. We cannot realistically handle millions of votes for each action in layer 1. Is randomized voting worse that requiring a large deposit to become a DRep?

If you have little stake, what is the real difference between having a low likelihood of being selected to be a DRep (where you will have much influence) vs casting a vote that is dwarfed by all the other votes? The difference is largely psychological, the theorems on random voting say things to the effect of "with very high probability, the randomized voting produces the same result".  But I understand that it might be psychologically unpalatable to some people.

It is my opinion/guess that a layer 2 solution would take a lot longer to become realized. We could, of course, try for a layer 2 solution after implementing what is proposed in this PR, with the caveat that we would be placing faith that the new government would accept such a plan.

Reply:
Could there be some off-chain solution made to batch dRep's voting preferences together in one tx? Or is the issue more with actually just counting that many votes?

Reply:
yea, that problem is getting that much information on chain, and processing it. I'm not dismissing a layer 2 solution, there is MUCH there to think about.

Reply:
Is there a case to be made that this CIP be implemented as is and some other CIP proposed if the number of active dReps becomes too high in the future? 

A point against that is if there are too many dReps causing too much congestion maybe the percent threshold to ratify anything couldn't even be reached.

Reply:
> A point against that is if there are too many dReps causing too much congestion maybe the percent threshold to ratify anything couldn't even be reached.

indeed, that's a real worry.

Reply:
I hope so too! There is certainly a lot of due diligence that we would need to do in order to be sure that the results apply to our particular setting, which is exactly what two of the IOG game theorists are looking at now.

Reply:
One comment on sortition: it has an "attention" component. If you are randomly selected to the committee of DReps for some decision, then your vote matters _and you had better be paying attention_. In government, you send someone round to their house and tell them they work for the government now, but on Cardano we could end up with situations where most of the selected DReps are "asleep" or something.

Reply:
I do indeed expect a lot of excitement at first, before folks realize that it is a lot of work to pay attention and vote to everything. The trouble is that these are all guesses...

Reply:
i would hope that the requirement to vote is rare. If the fundamentals are strong and the Costitution is consise Governace is mostly achieved already. Decentralised, Good means of deflationary exchange, digital Identity, globally accessable, AI oricals supplying consensus information on chain,  these attributes alone if preserved and maintained change everything.

Reply:
One idea I thought is about the loss of seed phrases over the next 500 years eventually degrading supply. What about validate your ADA yearly or some philosophically of phsycolocically benificial period or it's distributed accross all Cardano Wallets. Constitutional value- The Network is protected from loss  and distributed.

Reply:
> i would hope that the requirement to vote is rare

Some of the actions we expect to be rare, such as changing the constitution.

Some of the actions we expect to be common, such as treasury withdrawals for things like Catalyst.

Some of the actions could really go either way, there are valid arguments that some of the protocol parameters should be more adaptive.